en/pocket_privacy_policy_eu.md

```diff
@@ -0,0 +1,124 @@
+
+## User Accounts and Collection of Information {: #accounts }
+
+You are not required to create a user account or provide any personal information in order to visit the Pocket website. You are required to create a user account, however, to use our application. During registration, you are required to provide a name, password and valid email address.
+
+If you choose to share any articles, videos, or other content with another person, we will ask you to provide a valid email address for that person. We may also ask you to provide your username and password for third party websites in order to access the articles and information published on such sites. We also collect information if you contact us for support or help, submit an application or otherwise communicate with us. The information which you supply to us enables us to customize and enhance the services we provide to you. In addition, it helps us communicate with you about our news, events, promotions and new products.
+
+If you choose to subscribe to our paid subscription service, our third party payment processors will collect and handle your credit card and related payment information. Such third party processors’ use of your personal information is governed by their respective privacy policies.
+
+## Collection of Information as Part of Using Pocket Service {: #passive }
+
+In addition to the information that you provide to us when you register for a user account, we collect information about the URLs, titles and content of the web pages and other information you save to Pocket. The types of information we collect includes your browser type, device type, time zone, language, and other information related to the manner in which you access the Pocket Technologies. If you are on a mobile device, we collect the advertising identifiers provided by Apple on iOS and by Google on Android. You can change this identifier in your device settings. We also collect information about your use of the Pocket Technologies so that we can provide our services. For example, as a part of providing Pocket’s syncing features, we sync information about the items that you save and view within Pocket so that your list, tags, scroll position, and other account and usage information may be synced across all of your devices.
+
+## Use of Cookies and Analytics {: #cookies }
+
+### Types and Purposes
+
+We use four types of cookies and analytics tools for the following purposes:
+
+- Necessary: These cookies are necessary for our website to function. They cannot be switched off. You can set your browser to block or alert you about these cookies, but this may cause parts of the site to not work properly. These cookies do not store any personally identifiable information and unless Personalized Advertising Cookies are enabled, ads are non-personalized.
+- Preferences: We place these cookies on your device to remember your account preferences and language settings. Disabling these cookies may break some features.
```

Comment:
Is there a technical reason to put "Preferences" before "Analytics" compared to the US version of the doc?

Reply:
Thanks. Putting them in the same order, it might simplify translation.

Reply:
Agreed, thank you.